RFC-010-core-strategy/core-strategy.md

```diff
@@ -0,0 +1,65 @@
+# Executive Summary
+
+The Good Docs Project has grown organically since its creation, and has blossomed into a thriving community. Now it’s important to articulate where we are, how we can best support our users and community members, where we’d like to grow in future. 
+
+
+## Core Strategy Statement:  
+
+_The Good Docs Project educates and empowers people to create high quality documentation by providing them with resources, best practices and tools to enhance their documentation in open source and beyond._
+
+
+## What we do
+
+First and foremost, we create templates with supporting meta-documents that underpin the templates.
+
+As part of this work, we engage in many other initiatives around docs advocacy, docs education, and docs tooling. 
+
+We produce content to address the goals of engaging in our community and helping run the project; and learning and gaining experience in the realm of tech writing.
+
+
+### We don’t compete, we cooperate
+
+We don't compete with other groups and we aren't a business. We are one open source community looking to make open source, as a whole, better for everyone. 
```

Comment:
```suggestion
We don't compete with other groups and we aren't a business. We are one open-source community looking to make open source, as a whole, better for everyone. 
```
AFAIK, if used as an adjective, it is good practice to hyphenate open-source. 
Feel free to disregard if that's not relevant to the RFC.

Reply:
Google style guide recommends "open source", (which is apparently decided upon after much internal debate).
https://developers.google.com/style/word-list#open-source

Microsoft suggestions: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source

Reply:
Thanks @kickoke for picking this up. I normally would do that! 
Thanks for pulling those references @camerons - that's a bold move by Google to take that stance. I can see why they did it. 
I'm making an executive decision to go _with_ the hyphen.